ci_framework/roles/edpm_build_images/tasks/run.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Build EDPM hardened uefi image
+  become: true
```

Comment:
Is this intended?

Reply:
Yes, when someone install edpm-image-builder from rpm, its get installed in /usr/share/edpm-image-builder/ directory and the way https://github.com/openstack-k8s-operators/edpm-image-builder/blob/main/Containerfile.image is written its look for copy_out.sh/qcow2 in the same directory. 

For non rpm installation cases we don't need this become true, i have parameterized it here

become: "{{ cifmw_edpm_build_images_via_rpm }}"

i have already informed @steveb about this limitation of the Containerfile.image and we plan to improve the Containerfile.image in future, till then this is needed for rpm installation cases.